lib/parsers.js

```diff
@@ -95,7 +95,7 @@ module.exports = function(nanomatch, options) {
       if (!m) return;
       var val = m[0];
 
-      var isNegated = val.length % 2 = 1;
+      var isNegated = val.length % 2 === 1;
```

Comment:
thanks for clearly demonstrating the value of modules by screwing up something this simple.